examples/ckb-indexer-collector.ts

```diff
@@ -2,22 +2,42 @@
 
 import { Script, Indexer, helpers, config, BI } from "@ckb-lumos/lumos";
 
-config.initializeConfig(config.predefined.AGGRON4);
+config.initializeConfig(config.predefined.LINA);
 
-const CKB_RPC_URL = "https://testnet.ckb.dev/rpc";
-const indexer = new Indexer(CKB_RPC_URL);
+const nodeUri = "https://testnet.ckb.dev/rpc";
+const indexUri = "https://testnet.ckb.dev/indexer";
+const indexer = new Indexer(indexUri, nodeUri);
```

Comment:
it might be better if we can add another example to demonstrate exact search instead of modifying original example